38/upgrade.html

```diff
@@ -28,7 +28,8 @@ <h5><a id="upgrade_380_notable" href="#upgrade_380_notable">Notable changes in 3
             If READ is not authorized, checkpointing is limited to offsets mirrorred after the start of the task.
             See <a href="https://issues.apache.org/jira/browse/KAFKA-15905">KAFKA-15905</a> for more details.
         </li>
-        <li>Tiered Storage now supports clusters configured with multiple log directories (i.e. JBOD feature).</li>
+        <li>JBOD in KRaft is now production ready.</li>
+        <li>Tiered Storage, which is still in early availability, now supports clusters configured with multiple log directories (i.e. JBOD feature).</li>
```

Comment:
@stanislavkozlovski and @soarez I updated the `upgrade.html` file as well in here so the docs are fixed. If you like this I'll update https://github.com/apache/kafka/pull/16654 accordingly

Reply:
Slight preference for using the "early access" term instead of "production ready", like in my suggestion in https://github.com/apache/kafka-site/pull/614#discussion_r1687161313 as I suspect there's less misunderstanding that way.

But I think this is OK. Thanks @jlprat

Reply:
Hi @soarez I think you are looking at an outdated version of the text. The new one states:
> <li>Tiered Storage, which is still in early access, now supports clusters configured with multiple log directories (i.e. JBOD feature).</li>

Reply:
done